utils_cv/detection/data.py

```diff
@@ -132,3 +140,339 @@ def coco_labels() -> List[str]:
         "hair drier",
         "toothbrush",
     ]
+
+
+def extract_masks_from_labelbox_json(
+    labelbox_json: Union[str, Path],
+    old_dir: Union[str, Path],
```

Comment:
Variable name could be more informative. E.g. old_dir -> data_dir, new_dir-> mask_data_dir

Reply:
Changed